ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology-mpls.md

```diff
@@ -0,0 +1,192 @@
+# Fabric Topology Variables for MPLS Design
+
+The fabric topology variables define the connectivity between the various node types, as well as override the default switch properties.
+
+- The MPLS design supports any fabric topology variables already supported by l3ls-evpn, barring the exceptions outlined in this document.
+- The MPLS design additionally supports several new fabric topology variables that are outlined in this document.
+- Fabric Name, required to match Ansible Group name covering all devices in the Fabric | Required and **must** be an inventory group name.
```

Comment:
this should really be below the variables and options

Reply:
Moved it.